eZ/Publish/Core/IO/IOMetadataHandler/LegacyDFSCluster.php

```diff
@@ -153,11 +164,28 @@ public function exists($spiBinaryFileId)
     {
         $path = (string)$this->addPrefix($spiBinaryFileId);
 
-        $stmt = $this->db->prepare('SELECT name FROM ezdfsfile WHERE name_hash LIKE ? and mtime > 0 and expired != 1');
-        $stmt->bindValue(1, md5($path));
-        $stmt->execute();
-
-        return $stmt->rowCount() == 1;
+        $qb = $this->db->createQueryBuilder();
```

Comment:
I would rather prefer explicit naming, like `queryBuilder`, but that one has also its advantages.

Reply:
Renamed as per request :)